src/test/java/teammates/test/cases/ui/browsertests/InstructorHomePageUiTest.java

```diff
@@ -250,7 +250,8 @@ public void testRemindActions(){
         homePage.goToPreviousPage(InstructorHomePage.class);
         
         ______TS("remind action: OPEN feedback session - inner button");
-        
+
+        homePage.waitForAjaxLoaderGifToDisappear();
```

Comment:
I took some time to think about your concern, but perhaps we should just attach waitForAjaxLoaderGifToDisappear() to every backwards navigation? 

My reasoning is, there isn't always a table to wait for, there will be pages where there will be no table and every develop (new or core) will have to figure out why some back navigation have tabular checks and some do not.

I believe waiting implicitly for awhile more is a small price to pay for the brittleness of the test.

Then again, if the tradeoff is worth it then we can just make sure that every test that is made, we check if there's a table to wait for and if there's any, we wait for table-something and just continue once it's out.

What do you think? Up for discussion since I'm kind of conflicted between the two anyway..